src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Emitter/CoreBindingHelpers.cs

```diff
@@ -409,7 +410,7 @@ void EmitBindImplForMember(MemberSpec member)
                         member,
                         member.Name,
                         sectionPathExpr: GetSectionPathFromConfigurationExpression(configKeyName),
-                        canSet: true,
+                        canSet: true, canGet: true,
```

Comment:
> canGet: true [](http://example.com/codeflow?start=37&length=13)

Please move this to its new line. 

Also, could you please explain what happen when having a property like `string MyProp { private get; set; }`?

Reply:
I am seeing you have added good tests, could you add one more case for a property like `public string StringProp { set; private get; } = "bar";`? Thanks!